rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -278,9 +279,6 @@ fn run_create_proposal_texts(cmd: CreateProposalTexts) -> Result<()> {
     let mut sns_proposal_text_paths = Vec::new();
 
     {
-        // Switch to the commit being released while we generate the proposal texts
-        let _commit_switcher = CommitSwitcher::switch(commit.clone())?;
```

Comment:
note to reviewer, we don't actually want to do this here.  The proposals can be created without changing branches, and this prevents the script user from testing fixes to those scripts while running this process.